src/app/redux/AppReducer.js

```diff
@@ -2,18 +2,19 @@ import { Map, OrderedMap } from 'immutable';
 import tt from 'counterpart';
 
 // Action constants
+export const appActions = {
```

Comment:
why this change? this breaks API: https://github.com/steemit/condenser/blob/master/src/app/components/elements/ConvertToSteem.jsx#L6

Reply:
i'd rather have it named something else, like actionConsts or something